Assets/MixedRealityToolkit/Inspectors/ServiceInspectors/BaseMixedRealityServiceInspector.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.﻿
+
+using System;
+using UnityEditor;
+
+namespace Microsoft.MixedReality.Toolkit.Editor
+{
+    public class BaseMixedRealityServiceInspector : IMixedRealityServiceInspector
```

Comment:
>  public class BaseMixedRealityServiceInspector : IMixedRealityServiceInspector [](start = 2, length = 79)

Nit: Can you add some doccomments for this base class?

Reply:
Ah, nevermind, I guess <inheritdoc> would probably be good enough here because you have it already on IMixedRealityServiceInspector

---
In reply to: [278680928](https://github.com/Microsoft/MixedRealityToolkit-Unity/pull/3773#discussion_r278680928) [](ancestors = 278680928)